packages/shared/util.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import Vue from 'vue'
 import semver from 'semver'
+import { config } from '@vue/test-utils'
```

Comment:
Not sure why, but this new import here breaks the IIFE. Would love some help tracking this down and fixing it. See #1553